pyproject.toml

```diff
@@ -1,67 +1,105 @@
-[tool.poetry]
+[project]
 name = "nominal"
 version = "1.26.0"
 description = "Automate Nominal workflows in Python"
 authors = [
-    "Alexander Reynolds <alex.reynolds@nominal.io>",
-    "Alexander Hill <alex.hill@nominal.io>",
-    "JP <jack@nominal.io>",
-    "Michael Bauer <michael@nominal.io>",
+    { name = "Alexander Reynolds", email = "alex.reynolds@nominal.io" },
+    { name = "Alexander Hill", email = "alex.hill@nominal.io" },
+    { name = "JP", email = "jack@nominal.io" },
+    { name = "Michael Bauer", email = "michael@nominal.io" },
+    { name = "Drake Eidukas", email = "drake@nominal.io" },
 ]
-readme = "README.md"
 license = "MIT"
-homepage = "https://nominal.io"
-repository = "https://github.com/nominal-io/nominal-client"
-include = ["README.md", "LICENSE", "CHANGELOG.md"]
+readme = "README.md"
 
-[tool.poetry.scripts]
-nom = 'nominal.cli:nom'
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
```

Comment:
need to add 3.13 to the matrix test strategy and mypy checks in the `justfile` if we're gonna advertise 3.13